addons/l10n_es_edi_tbai/models/ir_actions_report.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, _
+from odoo.exceptions import UserError
+
+class IrActionsReport(models.Model):
+    _inherit = 'ir.actions.report'
+
+    def _render_qweb_pdf(self, report_ref, res_ids=None, data=None):
+        if self._get_report(report_ref).report_name in ('account.report_invoice_with_payments', 'account.report_invoice'):
+            moves_without_signature = self.env['account.move'].browse(res_ids).filtered(
```

Comment:
might be ok (I don't know functionally), but it seems like this will be applied to all invoice, even if I have only a belgian company and I'm not  just installed l10n_es_ebi_tbai without installing it.

seeing the template below, it should probably also check a country code or l10n_es_tbai_is_required

Reply:
You are right, I've extended the condition with l10n_es_tbai_is_required, since it checks everything that is needed :)

As for the addition of the .pot file into .tx/config, you mind checking what I've done, I emulated from the other entries, but it's the first time I am adding a .pot from scratch, just to be sure I've done what is needed. (I am adding the pot file, because I wanted to add the new term to be translatable, but I've noticed there is just no .pot for the module, so I've exported the whole of it and added it.

Reply:
looks good to me for the .tx/config (I've never done it)

1 typo: should be l10n_es_edi_tbai.pot and not l10n_multilang.pot (and alphabetically it should be before multilang)